package/MDAnalysis/core/groups.py

```diff
@@ -2703,7 +2703,12 @@ def atoms(self):
         The :class:`Atoms<Atom>` are ordered locally by :class:`Residue` in the
         :class:`ResidueGroup`.  Duplicates are *not* removed.
         """
-        ag = self.universe.atoms[np.concatenate(self.indices)]
+        # If indices is an empty list np.concatenate will fail (Issue #1999).
```

Comment:
I added a comment and also changed the code slightly.

Reply:
Tbh I liked the try/except more.  The `try` branch is usually faster and is the more common code path.

Reply:
Okay, personally I dislike these try/except constructs. It always looks like you don't really know what is going on. But you are right. I just tested both codes and there is no performance difference.

Reply:
I'm also in favor of try/except. It's a very common idiom (both in MDAnalysis and in Python in general).

Reply:
Okay. try/except is also fine for me. I reverted the change.